docs/source/performance_benchmarks.md

```diff
@@ -0,0 +1,38 @@
+
+# Performance Benchmarks
+
+## Large Language Models**
+
+### Pretraining
+
+- The results in the table below show pre-training performance (using NeMo Framework 24.07) of various models on DGXH100, with FP8 precision.
```

Comment:
I think it is better to create another bullet for the test environments used in the run.
For example, 
- Container: nemo 24.07 (link to the container)
- System: DGX-H100